Public/Src/Engine/Scheduler/Artifacts/FileContentManager.cs

```diff
@@ -1974,139 +1974,116 @@ private async Task<bool> PrepareDirectoriesAsync(PipArtifactsState state, Operat
                 // we still are mandated to finish materializing if possible and eventually complete the materialization task.
                 using (operationContext.StartOperation(PipExecutorCounter.FileContentManagerPlaceFilesDuration))
                 {
-                    using (var sharedOpaqueOutputsWrapper = Pools.FileArtifactSetPool.GetInstance())
+                    for (int i = 0; i < state.MaterializationFiles.Count; i++)

```

Comment:
The only change in this file is basically to not compute `allSharedOpaqueOutputs` and not apply timestamps (what used to be line 2089).  The diff is big only because indentation changed (because there is one fewer `using` block now)

Reply:
Is the flagging of shared opaque outputs now taken care of by Scheduler's post process?

---
In reply to: [272297613](https://github.com/Microsoft/BuildXL/pull/83#discussion_r272297613) [](ancestors = 272297613)

Reply:
yes